src/addons/Pagination/PaginationItem.js

```diff
@@ -66,7 +66,7 @@ class PaginationItem extends Component {
 
   render() {
     const { active, ariaLabel, type, ...rest } = this.props
-    const disabled = type === 'ellipsisItem'
+    const disabled = rest.disabled || type === 'ellipsisItem'
```

Comment:
This sort of makes me wonder why `rest` props aren't spread _after_ all other props below, see lines 72-79.  Usually, the user's props should always "win".

@layershifter is there a reason for this or what is it an oversight?  Also, shouldn't the props passed to the `MenuItem` factory be passed as `{ defaultProps: { ... } }` so that they don't override the users `rest` props?

Reply:
Yep, it's my miss. I will check this PR on this weekend